src/server.py

```diff
@@ -45,7 +127,10 @@ async def redirect_link(request, link_endpoint):
                 )
             )
             url = await query.fetchone()
-            return response.redirect(url[1])
+            if url[3]:
```

Comment:
Isn't it safer to use something like this?

```suggestion
            if url.is_active:
```